Incorrect Application Channel Name

MothTheMortal opened the pull request:
Put the correct channel name for applications in the code, resolves #16.

MothTheMortal committed:
Incorrect Application Channel Name

Put the correct channel name for applications in the code.

MothTheMortal committed:
Fixed Application Log Name

Fixed the incorrectly put applications log name in the code.

StarbuckBarista reviewed (changes requested):
While you have done this correctly, there are multiple instances where `applications` is used instead of `application_logs`. I believe that there are three, but you will need to be the one who searches through the code and updates all of them.

StarbuckBarista commented on src/python/cogs/applications.py:
```diff
@@ -607,7 +607,7 @@ async def reject_application(last_reply, note=None):
                 log_embed.set_footer(text="The user did not receive a notification because their DMs are disabled.")
 
-            log_channel = self.client.get_channel(config.channel_ids["applications"])
+            log_channel = self.client.get_channel(config.channel_ids["application-logs"])
```
This should be `application_logs`, rather than `application-logs`.

StarbuckBarista reviewed (changes requested):
Also just noticed that the config key is incorrect here.

MothTheMortal committed:
Updated Application Log Names

Updated application log names in the code, as messages were being sent in the wrong channel.

StarbuckBarista reviewed (approved):
LGTM!
